src/pages/my_page/user_profile/UserRoles.tsx

```diff
@@ -86,6 +90,20 @@ export const UserRoles = ({ user, hasActiveEmployment }: UserRolesProps) => {
           text={t('my_page.roles.nvi_curator_description')}
         />
       )}
+      {isThesisCurator && (
+        <RoleItem
+          dataTestId="user-role-thesis-curator"
+          label={t('my_page.roles.thesis_curator')}
+          text={t('my_page.roles.thesis_curator_description')}
+        />
+      )}
+      {isEmbargoThesisCurator && (
+        <RoleItem
+          dataTestId="user-role-embargo-thesis-curator"
+          label={t('my_page.roles.thesis_embargo_curator')}
+          text={t('my_page.roles.thesis_embargo_curator_description')}
+        />
```

Comment:
Usikker på om denne skal indenteres for å vise rolle-avhengighet. Venter på avklaring